adding_metadata_betydb_yaba.md

```diff
@@ -0,0 +1,28 @@
+## BETYdb-YABA and Python Client Library
+
+BETYdb-YABA and its python client library provides an automated bulk upload to many BETYdb tables. API endpoints have been implemented to upload data to respective tables. For more information, see the BETYdb-YABA [README](https://github.com/PecanProject/BETYdb-YABA/blob/master/README.md) and [swagger documentation](https://app.swaggerhub.com/apis-docs/y51/bet-ydb_yaba/1.0.5).
```

Comment:
External links should open in a new tab: so use `[README](https://github.com/PecanProject/BETYdb-YABA/blob/master/README.md){target="_blank"}` in place of just `[README](https://github.com/PecanProject/BETYdb-YABA/blob/master/README.md)`.